docs/website/docs/getting-started/pytorch.md

```diff
@@ -0,0 +1,100 @@
+# PyTorch Integration
+
+IREE supports compiling and running PyTorch programs represented as
+`nn.Module` [classes](https://pytorch.org/docs/stable/generated/torch.nn.Module.html) as well as models defined using [`functorch`](https://pytorch.org/functorch/).
+
+## Prerequisites
+
+Install IREE pip packages, either from pip or by
+[building from source](../building-from-source/python-bindings-and-importers.md):
+
+```shell
+pip install \
+  iree-compiler \
+  iree-runtime \
+  iree-tools-tf
+```
+
+Install [`torch-mlir`](https://github.com/llvm/torch-mlir), necessary for compiling PyTorch models to a format IREE is able to execute:
+
+```shell
+pip install -f https://llvm.github.io/torch-mlir/package-index/ torch-mlir
+```
+
+A special `iree_torch` package is available to make it easy to compile PyTorch programs and run them on IREE:
+
+```shell
+pip install git+https://github.com/iree-org/iree-torch.git
+```
+
+## Running a model
+
+Going from a loaded PyTorch model to one that's executing on IREE happens in four steps:
+
+ 1) Compile the model to [MLIR](https://mlir.llvm.org)
+ 2) Compile the MLIR to IREE VM flatbuffer
+ 3) Load the VM flatbuffer into IREE
+ 4) Execute the model via IREE
+
+!!! note In the following steps, we'll be borrowing the model from [this BERT colab](https://github.com/iree-org/iree-torch/blob/main/examples/bert.ipynb) and assuming it is available as `model`.
+
+### Compile the model to MLIR
+
+First, we need to trace and compile our model to MLIR:
+
+```python
+model = # ... the model we're compiling
+example_input = # ... an input to the model with the expected shape and dtype
+mlir = torch_mlir.compile(
+    model,
+    example_input,
+    output_type=torch_mlir.OutputType.LINALG_ON_TENSORS,
+    use_tracing=True)
+```
+
+The full list of available output types can be found [here](https://github.com/llvm/torch-mlir/blob/6403c0e56f0e93e231df1c8d3dc78df7dd721b80/python/torch_mlir/__init__.py#L19) and includes linalg on tensors, mhlo, and tosa.
+
+### Compile the MLIR to IREE VM flatbuffer
+
+Next, we compile the resulting MLIR to IREE VM flatbuffer: 
+
+```python
+iree_backend = "llvm-cpu"
+iree_vmfb = iree_torch.compile_to_vmfb(mlir, iree_backend)
+```
+
+Here we have a choice of backend we want to target.  See [IREE Deployment Configurations](https://iree-org.github.io/iree/deployment-configurations/) for a full list of targets.
```

Comment:
Nit: no double space after period.

Reply:
In theory markdown should be readable in plaintext format, where it's often done in monospace fonts.  The double space after the period helps distinguish between end of sentence vs splitting words in those cases.  I think it's a bit of a relic from the typewriter days, but for the case like this where we expect it to potentially still be viewed monospace I think it makes sense.

That being said, if the other markdown files aren't following suit, I should just be consistent.  I've updated it.